From 0fbdc62eafc7b3ef34ea6e0231e7b1a1430d2cd8 Mon Sep 17 00:00:00 2001
From: Brian Bennett <brian.bennett@joyent.com>
Date: Fri, 26 Aug 2016 13:48:55 -0700
Subject: [PATCH] OS-5634 system-log SMF method has existential confusion
 Reviewed by: Alex Wilson <alex.wilson@joyent.com> Reviewed by: Josh Wilsdon
 <josh.wilsdon@joyent.com>

---
 overlay/generic/lib/svc/method/system-log | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/overlay/generic/lib/svc/method/system-log b/overlay/generic/lib/svc/method/system-log
index f8a60a36..c2424870 100644
--- a/overlay/generic/lib/svc/method/system-log
+++ b/overlay/generic/lib/svc/method/system-log
@@ -103,20 +103,21 @@ fi
 
 if [ -f /etc/rsyslog.conf ]; then
 	if [ ! -x /usr/sbin/rsyslogd ]; then
-		echo "/etc/rsyslog.conf is missing. Exiting."
+		echo "Executable /usr/sbin/rsyslogd is not found. Exiting."
 		exit $SMF_EXIT_ERR_CONFIG
 	fi
 else
 	#
 	# Fall back to old syslogd
 	#
+	echo "/etc/rsyslog.conf is missing. Using syslogd instead."
 	if [ ! -f /etc/syslog.conf ]; then
 		echo "/etc/syslog.conf is missing. Exiting."
 		exit $SMF_EXIT_ERR_CONFIG
 	fi
 
 	if [ ! -x /usr/sbin/syslogd ]; then
-		echo "Executable /usr/sbin/syslogd not found. Exiting"
+		echo "Executable /usr/sbin/syslogd not found. Exiting."
 		exit $SMF_EXIT_ERR_CONFIG
 	fi
 fi
-- 
2.21.0

